tutorials/meshing_of_fractures.ipynb

```diff
@@ -280,19 +287,23 @@
    "metadata": {},
    "source": [
     "We have not yet set a boundary for the `FractureNetwork3d`, and effectively for\n",
-    "the domain. The boundary is defined as a box, and is imposed in the following way:"
+    "the domain. In the previous examples the bounding box was given on instantiation of the fracture network. However, it can also be imposed at a later point, in the following way:"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
-   "metadata": {},
+   "execution_count": 7,
+   "metadata": {
+    "scrolled": true
+   },
    "outputs": [
     {
      "data": {
-      "text/plain": "array([0, 1, 2])"
+      "text/plain": [
```

Comment:
What's happening here?

Reply:
It seems like it is printing the output of the impose_external_boundary method for some reason. The problem seems specific to jupyter notebook, because nothing is printed out if I run the same script as a Python file.